.gitignore

```diff
@@ -84,6 +84,8 @@ Makefile
 /TAGS
 /libcrypto.map
 /libssl.map
+/libcrypto.ld
+/libssl.ld
```

Comment:
I would make that `*.ld`, and while we're at it, we should probably also ignore `*.map`

Reply:
Done

Reply:
For future reference, not just in root.

Reply:
Are you asking me to change .gitignore, to ignore those files anywhere? .ld files and .map files? Why might they appear anywhere?

Reply:
Yes.
Think future modules that we want to make similar ld scripts for. Say we'd like to do that for engines (we really should). Where are those ld scripts going to end up?

Reply:
Updated again